docs/index.md

```diff
@@ -37,8 +37,6 @@ val driver: SqlDriver = AndroidSqliteDriver(Database.Schema, context, "test.db")
 dependencies {
   implementation "com.squareup.sqldelight:native-driver:1.2.1"
 }
-
-// You'll also need to have SQLite linked via -lsqlite3 during compilation.
```

Comment:
I was thinking that this is already enabled by default with the `linkSqlite` gradle property. Having this comment here without a link for more instructions might be confusing? Please correct if this is wrong.

Reply:
Approving everything but this because I know nothing about it. @AlecStrong or @kpgalligan.

Reply:
Fwiw the `linkSqlite` flag being true by default is mentioned in [gradle.md](https://cashapp.github.io/sqldelight/gradle/): 

![image](https://user-images.githubusercontent.com/2387680/71121811-2f02e880-21ad-11ea-9444-3f36fb9a94ee.png)

Reply:
yup this makes sense